res/skins/Deere/left_deck_mixer_controls.xml

```diff
@@ -14,19 +14,16 @@
       <!-- In 2 deck mode, we show a 3-column mixer section -->
       <WidgetGroup>
         <Layout>horizontal</Layout>
+        <SizePolicy>min,min</SizePolicy>
         <Children>
+          <Template src="skin:deck_mixer_controls_col1_left.xml"/>
           <WidgetGroup>
             <Layout>horizontal</Layout>
             <Children>
-              <Template src="skin:deck_mixer_controls_col1_left.xml"/>
+              <Template src="skin:deck_mixer_controls_col2.xml"/>
+              <Template src="skin:deck_mixer_controls_col3.xml"/>
             </Children>
-            <Connection>
-              <ConfigKey>[Master],show_eqs</ConfigKey>
-              <BindProperty>visible</BindProperty>
-            </Connection>
           </WidgetGroup>
-          <Template src="skin:deck_mixer_controls_col2.xml"/>
-          <Template src="skin:deck_mixer_controls_col3.xml"/>
```

Comment:
columns2/3 were squeezed and set off to the left, column1 was half-covered.
Don't know why THIS works..
Reproduce:
* 2 decks
* split waveforms
> with skin settings turn EQs off, then on again